patterns/vsi-extension/README.md

```diff
@@ -1,27 +1,16 @@
 # Add a VSI to a landing zone VPC
 
-This logic creates a VSI to an existing landing zone VPC.
+## Before you begin
 
-This code creates and configures the following infrastructure:
-- Adds an SSH key to IBM Cloud or uses an existing one.
-- Adds a VSI in each subnet of the landing zone VPC.
+- The block storage to KMS auth policy must exist before provisioning this DA. This policy would have been created by the VPC landing zone DA if the `add_kms_block_storage_s2s` option is set to true, which is set to `true` by default.
 
-There are two ways through which a user can pass the VPC details for deploying the VSI, both the approaches are mutually exclusive.
+To deploy this architecture, you need the VPC ID, subnet names, and boot volume encryption key from your existing VPC landing zone deployable architecture. For information about finding the information, see [Adding a VSI to your VPC landing zone deployable architecture](https://cloud.ibm.com/docs/secure-infrastructure-vpc?topic=secure-infrastructure-vpc-ext-with-vsi).
 
-## Using `vpc_id`
+This extension DA can be provisioned ontop of the VPC landing zone DA, as well as the ROKS landing zone DA since both deploy an SLZ VPC.
 
-The VPC ID of the landing zone VPC can be assigned to the variable vpc_id in order to create a VSI within that specific VPC.
+This architecture creates a VSI in some or all of the subnets of your existing landing zone VPC deployable architecture.
 
-## Using `prerequisite_workspace_id` and `existing_vpc_name`
+![Architecture diagram for adding a VSI to your VPC landing zone deployable architecture](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/main/reference-architectures/vsi-extension.drawio.svg)
 
-The user can specify the workspace ID associated with the deployment of the landing zone VPC when creating a new VSI.
```

Comment:
@SirSpidey since we are removing the input for workspace, users who have deployed the VPC pattern via EPX/Project/Schematic/Catalog might need to know how to lookup some of the existing values that need to be supplied. We have some instruction links in the descriptions of the individual input variables, should we have something at this README level as well?

Reply:
@toddgiguere we probably want to add the following kind of information to the `architecture overview` tab in the DA catalog ([like this](https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-pvs-inf-2dd486c7-b317-4aaa-907b-42671485ad96-global?catalog_query=aHR0cHM6Ly9jbG91ZC5pYm0uY29tL2NhdGFsb2cjcmVmZXJlbmNlX2FyY2hpdGVjdHVyZQ%3D%3D&kind=terraform&format=terraform&version=8e349dfd-a44d-413a-9aba-e7a228fd583e-global) for powervs with vpc extension). 

> To deploy this architecture, you need the VPC ID, subnet names, and boot volume encryption key from your existing VPC landing zone deployable architecture. For information about finding the information, see [Adding a VSI to your VPC landing zone deployable architecture](https://cloud.ibm.com/docs/secure-infrastructure-vpc?topic=secure-infrastructure-vpc-ext-with-vsi).

This content is in the extension readme in this pr

Reply:
@ocofaigh this comment above is more likely a comment for you, or whoever is onboarding the extension. I don't think the architecture overview tab is populated from the catalog json.